batch/test/test_batch.py

```diff
@@ -20,8 +20,8 @@
 
 DOCKER_PREFIX = os.environ.get('DOCKER_PREFIX')
 DOCKER_ROOT_IMAGE = os.environ.get('DOCKER_ROOT_IMAGE', 'gcr.io/hail-vdc/ubuntu:18.04')
-DOMAIN = os.environ['DOMAIN']
-NAMESPACE = os.environ['HAIL_DEFAULT_NAMESPACE']
+DOMAIN = os.environ.get('DOMAIN')
```

Comment:
Can you explain this change to me?

Reply:
The `test_batch_invariants` test was failing because of a missing environment variable, which seemed odd to me until I realized that `test_batch_invariants` doesn't define *any* environment variables. Making these optional allowed for a test that didn't need them to omit them in `build.yaml`.

Reply:
Is this a pytest issue where it runs all the code in that folder and then runs the specified test? If so, do you think this would work better?

```
python3 -m pytest --log-date-format="%Y-%m-%dT%H:%M:%S" --log-format="%(asctime)s %(levelname)s %(name)s %(filename)s:%(lineno)d:%(funcName)s %(message)s" --log-cli-level=INFO -s -vv --instafail /io/test/test_invariants.py
```

Reply:
Ya I think that could work. Could I do a separate PR where we put that test in its own file and then require that the environment variables for a test file be provided?